### ISSUE TITLE
More than 200 labeled images are used to train

### ISSUE THREAD

**Issue by yongpi-scu** [LDOUBLEV/semi-supervised-GAN#6]
Hi,
Thanks for your work.
I think there is something wrong in your result.In general, we want to use only 200 labeled images in training process.But in your code, you adopt the mnist dataloader provide by tensorflow. 
https://github.com/LDOUBLEV/semi-supervised-GAN/blob/52ba4a5ebb5ff07e6eceb48897ca78cb6b14f529/train.py#L120
In [mnist.train.next_batch()](https://github.com/tensorflow/tensorflow/blob/0f1894de6cfddb3df8a3d6865d97b947af56f995/tensorflow/contrib/learn/python/learn/datasets/mnist.py#L183) function, there is a default param `shuffle=True`,so each epoch used different 200 labeled images.Consider overlapping, the number of labeled  images is in [200, 200*epochs].
Am i understanding right? Looking forward to your reply.
Thank you.

**Comment by LDOUBLEV** [comment]
@PYsaber Hi, thanks for your question.
I set a flag to choose whether the label information works in the loss function of the network.You can check the code include 70thline and 119th line in the train.py. 
The label information are not used in the loss function when self.flag is equal 0.
